code-challenges.js

```diff
@@ -9,37 +9,100 @@
 // --------------------1) Create a function that takes in a number and determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp1 = 35
-var temp2 = 350
-var temp3 = 212
+// Create a function that determines if a numbers is below, at, or above boiling point
+// Boiling point is 212 degrees farenheit
+// Input: [35, 350, 212]
```

Comment:
i'm glad you are describing the input, but in this case the input will be the test variables provided on lines 17-19